### DISCUSSION TITLE
What does `affinity` mean

### DISCUSSION THREAD

**Discussion by Dante-dan** [ianstormtaylor/slate#4860] [Q&A]
I cann't understand [`affinity` option](affinity) , when I read docs about `PointRef`, `RangeRef`.

At the beginning, I think `affinity` and `direction` are similar in meaning. But now I think it may be wrong.

May you give me some examples to understand it？ 🙏

**Comment by dylans** [comment]
Maybe this recent PR might help a bit: https://github.com/ianstormtaylor/slate/pull/4848 ?
